recipes/aws-lambda-cpp/all/test_package/CMakeLists.txt

```
@@ -6,3 +6,5 @@ find_package(aws-lambda-runtime REQUIRED CONFIG)
 add_executable(${PROJECT_NAME} test_package.cpp)
 target_link_libraries(${PROJECT_NAME} PRIVATE AWS::aws-lambda-runtime)
 target_compile_features(${PROJECT_NAME} PRIVATE cxx_std_11)
+
+aws_lambda_package_target(${PROJECT_NAME})
```

According to the actions taken in other test_packages, we try to simplify them as much as possible any trouble maintaining them. Also, we had a similar case with protobuf here https://github.com/conan-io/conan-center-index/blob/master/recipes/protobuf/all/test_package/CMakeLists.txt#L28-L30, so let's follow the same path here as well 😄 

```suggestion

if(NOT COMMAND aws_lambda_package_target)
    message(FATAL_ERROR "aws_lambda_package_target should have been defined as part of find_package(aws-lambda-runtime)")
endif()
```

---

@danimtb 
Thanks a lot for your suggestion!!
I merged it.